### DISCUSSION TITLE
playing mp3 file by name from SD

### DISCUSSION THREAD

**Discussion by Gendiaaa** [pschatzmann/arduino-audio-tools#1089] [Q&A] [answered]
hello,
  i am trying to make simple voice reminder with esp32 and i was able to play the files loaded in SD card and output as i2s from esp32 with no problem but i want to run specific sound file by name the library the player.begin(index) method is calling the file by index !
is there are any way to play and request file by name ?

**Comment by pschatzmann** [comment, marked as answer]
Yes, there is: check the class documentation!
